File: app/src/main/java/com/luigi573/mira/statistics/StatisticsActivity.java
package com.luigi573.mira.statistics;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.luigi573.mira.R;
import com.luigi573.mira.base.BaseActivity;
import com.luigi573.mira.util.ActivityUtils;

import javax.inject.Inject;

public class StatisticsActivity extends BaseActivity {


    @Inject StatisticsFragment fragment;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.statistics_act);
        ActivityUtils.addFragmentToActivity(getSupportFragmentManager(), fragment, R.id.container);



    }
}

File: app/src/main/java/com/luigi573/mira/statistics/StatisticsActivity.kt
package com.luigi573.mira.statistics

import android.os.Bundle
import android.os.PersistableBundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.add
import androidx.navigation.findNavController
import com.luigi573.mira.R

class StatisticsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.statistics_act)
//        val fragment =StatisticsFragment()
//        val fragmentManager = supportFragmentManager
//        fragmentManager.beginTransaction().add(R.id.container, fragment).commit()
//        val navController = findNavController(R.id.nav_host_fragment)
//        navController.navigate(R.id.statisticsFragment)
    }
}